condDatasetSubmitter.py

```diff
@@ -423,6 +435,9 @@ def createCMSSWConfigs(options,confCondDictionary,allRunsAndBlocks):
             driver_command += '--customise %s ' % (details['customise'])
         if details['era'] != "":
             driver_command += "--era %s " % (details['era'])
```

Comment:
hello @bajarang 
please Xcheck one by one
all these checks on the content of details dictionary